README.md

```diff
@@ -204,11 +204,11 @@ julia> using BenchmarkTools
 
 julia> 
 @btime logpdf($m2(X=X), $truth)
-  802.533 ns (16 allocations: 464 bytes)
+  14.209 μs (52 allocations: 1.25 KiB)
```

Comment:
@thautwarm I think things are looking good except for this performance regression. I'll try re-running locally, maybe it was just something with my system

Reply:
Looks like it's because `logpdf` is dispatching dynamically, should have it fixed soon

Reply:
I think we can avoid using parameteric `@gg` when the model is Soss, could we do some dispatch on `rand`?

Reply:
I'm now facing something disturbing(about my graduate education stuffs), so I cannot make the optimizations immediately.
However my personal recommendation is to keep efficienct. I'd request for changes, and make a PR after settling the urgent things.
If you feel urgent to merge this, it's still okay.

Reply:
I agree, no merge until this is settled.

I had thought it might be due to the new GG approach, but now I think there may be overhead in the function I'm wrapping it with. Need to do some benchmarking to be sure.

No rush here, happy for your school to take first priority in general. We can chat when you get past the current situation. Best of luck :)

Reply:
https://github.com/cscherrer/Soss.jl/issues/65